src/constants/constants.js

```diff
@@ -71,3 +72,13 @@ export const MAP = {
         backgroundColor: "grey"
     }
 };
+
+export const calcLevel = function(xp, lv = 1) {
```

Comment:
Not sure if this is the appropriate place for this. Should we make a utils file for misc methods? Thoughts?

Reply:
Yeah that sounds good but I'm not sure what else would go there as of now

Reply:
Gonna wait til we have more related utilities then I'll separate it out.